src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -311,7 +311,7 @@ private LoggerConfiguration CreateLoggerConfiguration(IServiceProvider servicePr
             
             return new LoggerConfiguration()
                 .MinimumLevel.Debug()
```

Comment:
Is it a good idea to set the loglevel to Debug ?  I mean, Arcus is a library, customers who use this library in their project might get overwhelmed with Arcus logs that they do not want.

Reply:
Or set it on Debug when the configuration is Debug 😄 .

Reply:
+1. I'd use info by default

Reply:
I'd set it to Info, regardless if the configuration is debug or release.